app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -73,7 +74,21 @@ private fun setupGlean(context: Context) {
     Glean.initialize(context, Configuration(httpClient = lazy { context.components.core.client }))
     Glean.setUploadEnabled(BuildConfig.TELEMETRY_ENABLED && Settings.isTelemetryEnabled(context))
     GleanFactProcessor().register()
-    Experiments.initialize(context, ExperimentsConfiguration(httpClient = lazy { context.components.core.client }))
+    Experiments.initialize(
+        context,
+        ExperimentsConfiguration(httpClient = lazy { context.components.core.client })
+    )
```

Comment:
Quick question about the initialization (unrelated to the PR though): Is it planned that the experiments of all apps will live in the `mobile-experiments` collection?

Reply:
That's a good question and not one I have the answer to 😞 .  Right now that's certainly the case as we have only deployed schema for that particular collection name.  It might be worth discussing it with Benson's team who will be owning this soon (or maybe @georgf knows and can tell us when he's back from PTO)

Reply:
The collection name `mobile-experiments` [appears to be hard coded](https://github.com/mozilla-mobile/android-components/blob/d5568f2776e5dbe68f93d6eb2fd4494cc5605b43/components/service/experiments/src/main/java/mozilla/components/service/experiments/ExperimentsUpdater.kt#L148-L149) in the experiments component. So yes, all experiments of all apps will live in the mobile-experiments collection. I also don't believe we have a way to distinguish between different apps in experiments.

We'll definitely want to improve this in the future.

Reply:
I looked at this more closely (after familiarizing myself a bit more with the library) and what I said isn't quite right. Although the collection name is hard coded, It seems there *is* a way to distinguish different apps. Specifically, we can filter experiments based on their package name.

Reply:
Yes, [`match.app_id`](https://github.com/mozilla-mobile/android-components/tree/master/components/service/experiments#experiment-fields) can be used to select specific applications, using a regex on their application id.